apps/nextjs/src/components/DialogControl/ContentOptions/ClearSingleChatFromChatHistory.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { useTransition } from "react";
+import { toast } from "react-hot-toast";
+
+import {
+  OakFlex,
+  OakP,
+  OakSmallPrimaryButton,
+  OakSmallSecondaryButton,
+} from "@oaknational/oak-components";
+import { showReportDialog } from "@sentry/nextjs/build/types/server";
+
+import LoadingWheel from "@/components/LoadingWheel";
+import { trpc } from "@/utils/trpc";
+
+const ClearSingleChatFromChatHistory = ({
+  closeDialog,
+}: {
+  closeDialog: () => void;
+}) => {
+  const { mutate, isLoading } =
+    trpc.chat.appSessions.deleteChatById.useMutation({
+      onSuccess() {
+        console.log("Chat history cleared");
+        closeDialog();
+        localStorage.removeItem("chatIdToDelete");
```

Comment:
We've seen localStorage failures on Oak when it's not available on some teacher computers. Maybe try / catch this